docs/sql-data-sources-load-save-functions.md

```diff
@@ -41,6 +41,11 @@ name (i.e., `org.apache.spark.sql.parquet`), but for built-in sources you can al
 names (`json`, `parquet`, `jdbc`, `orc`, `libsvm`, `csv`, `text`). DataFrames loaded from any data
 source type can be converted into other types using this syntax.
 
+The available extra options are documented in the API documentation. For basic formats
+(like `json`, `parquet`, `jdbc`, `orc`, `csv`, `text`) you want to check the API documentation of
+`org.apache.spark.sql.DataFrameReader` and `org.apache.spark.sql.DataFrameWriter`. For other
```

Comment:
BTW, this wouldn't cover Pyspark and Sparks users. Let's generalize it.

Reply:
OK, I agree it should cover other languages as well.

Could you please list the places one would look at for Pyspark and Sparks, please? Then I can do the legwork of formatting it nicely and updating the pull request.

Reply:
You can link to, for example, http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.sql.DataFrameWriter and http://spark.apache.org/docs/latest/api/python/pyspark.sql.html?highlight=dataframewriter#pyspark.sql.DataFrameWriter I think the Scala docs are probably most comprehensive though, and would be relevant to Pyspark users too.

Reply:
Yup. I think we can just simply say,

Please refer API documentation for available options in built-in sources, for example, `org.apache.spark.sql.DataFrameReader` and `org.apache.spark.sql.DataFrameWriter`.

than making things complicated.

Reply:
I think we are reaching a good compromise here: there is a good and usable hint regarding where to start, but not too much details. I've updated the documentation according to your suggestions